code/modules/power/apc/apc_main.dm

```diff
@@ -109,13 +109,21 @@
 	var/datum/alarm_handler/alarm_manager
 	/// Offsets the object by APC_PIXEL_OFFSET (defined in apc_defines.dm) pixels in the direction we want it placed in. This allows the APC to be embedded in a wall, yet still inside an area (like mapping).
 	var/offset_old
-	var/cuttedAIwire = FALSE
+	/// Used for apc helper called cutted_AI_wire to make apc's wore responsible for ai connectione mended.
+	var/cutted_AI_wire = FALSE
```

Comment:
`cut_AI_wire` would be better grammatically. sorry if i didn't get that across the first time.